tools/errortracker/errortracker.go

```diff
@@ -99,13 +99,17 @@ func handle(w http.ResponseWriter, r *http.Request) {
 
 	// Fill query params into JSON struct.
 	line, _ := strconv.Atoi(r.URL.Query().Get("l"))
+	errorType := "default"
+	if r.URL.Query().Get("a") == "1" {
```

Comment:
That the error came from an assert. Thought it was obvious given the thing that happens next.